### PR TITLE
Allow to explicitly set the type of a change recommendation

### PR DESCRIPTION
**Pull request by CatoTH**

### CODE REVIEW

File: openslides/motions/static/js/motions/site.js [hunk 1]
```diff
@@ -285,7 +285,23 @@ angular.module('OpenSlidesApp.motions.site', [
     'gettextCatalog',
     'Editor',
     'Config',
-    function(gettextCatalog, Editor, Config) {
+    'formlyConfig',
+    function(gettextCatalog, Editor, Config, formlyConfig) {
+        formlyConfig.setType({
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: Is this something generic? It looks so, so I propose to move it to a reusable factory or other service.

Comment (CatoTH) [inline openslides/motions/static/js/motions/site.js]: I moved the type definition to the site-wide formly-configuration in core/site.js to the other custom formly types.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Please review again.

**Comment by normanjaeckel** [comment]
I do not know whether @emanuelschuetze wants to test this again. If not please merge immediately.

**Comment by emanuelschuetze** [comment]
Retested. Ok.

**Pull request merged**